cli/build.rs

```diff
@@ -43,14 +43,6 @@ fn create_snapshot(
   println!("Snapshot written to: {} ", snapshot_path.display());
 }
 
```

Comment:
Higher up in this file the method `create_snapshot` calls 

```
  deno_web::init(&mut js_runtime);
  deno_fetch::init(&mut js_runtime);
  deno_crypto::init(&mut js_runtime);
```

Are these necessary for the compiler snapshot?

Reply:
Yes, declaration files are pulled into TSC during build

Reply:
I think only the `get_declaration()` methods are necessary. Maybe some of the event stuff from deno_web. But I don't think tsc calls fetch.

Reply:
I've removed `init` calls